File: c/1010.c
#include <stdio.h>

int main(){
    int a, b, a2, b2;
    double c, c2;

    scanf ("%d %d %lf", &a, &b, &c);
    scanf ("%d %d %lf", &a2, &b2, &c2);
    printf("VALOR A PAGAR: R$ %.2lf\n", (b*c) + (b2*c2));
}

File: js/1010.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var a = lines.shift().split(' ')
var b = lines.shift().split(' ')
console.log("VALOR A PAGAR: R$", ((parseFloat(a[1])*parseFloat(a[2]))+(parseFloat(b[1])*parseFloat(b[2]))).toFixed(2))
